redis/tests/test_types.rs

```diff
@@ -162,26 +197,26 @@ fn test_arc_slice() {
             Value::BulkString("2".into()),
             Value::BulkString("3".into()),
         ]));
-        assert_eq!(v, Ok(Arc::from(vec![1i32, 2, 3])));
```

Comment:
why is this change required?

Reply:
because you can have now `Arc<T>` and `Arc<[T]>` and you must give Rust a hint about type.

Reply:
this seems to break more tests than just this. if you remove the explicit support for `Arc<[T]>`, would it still be derived by the macros?

Reply:
it will not be delivered because implementation for `Arc<[T]>` is to create `Vec<T>` first and then use [Arc::from](https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-From%3CVec%3CT,+A%3E%3E-for-Arc%3C%5BT%5D,+A%3E) as far as I understand.

Reply:
But I accidentally changed testing `Arc<[T]>` into `Arc<Vec<T>>` I fixed this with [`4f1d21e` (#1088)](https://github.com/redis-rs/redis-rs/pull/1088/commits/4f1d21ea4c086e368e7420063e2d5ad9e266fce6) commit